seatunnel-api/src/main/java/org/apache/seatunnel/api/table/type/SeaTunnelRow.java

```diff
@@ -54,6 +54,10 @@ public void setRowKind(RowKind kind) {
         this.kind = kind;
     }
 
+    public void setKind(RowKind kind) {
```

Comment:
Should  be create `SeaTunnelRowWarpper` for flink engine adapter?

Reply:
`SeaTunnelRowWarpper` warp `SeaTunnelRow` object or extends SeaTunnelRow class

Reply:
I think no need to do this. Unified the data pojo is better.

Reply:
To maintain compatibility, we annotate the duplicate old methods with @Deprecated and explain why they cannot be deleted.

Reply:
add `@Deprecated` to `getRowKind` method
cc @TyrantLucifer

Reply:
update all caller `getRowKind `、`setRowKind` to  `getKind `、`setKind`

Reply:
cc @EricJoy2048 @Hisoka-X @Carl-Zhou-CN